types/cli-table/index.d.ts

```diff
@@ -1,45 +1,55 @@
 // Type definitions for cli-table 0.3
 // Project: https://github.com/Automattic/cli-table
 // Definitions by: AryloYeung <https://github.com/arylo>
+//                 Antoni Spaanderman <https://github.com/antonilol>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.1
+// TypeScript Version: 4.9
 
-interface TableOptions {
-    chars: Partial<Record<(
-        'top' |
-        'top-mid' |
-        'top-left' |
-        'top-right' |
-        'bottom' |
-        'bottom-mid' |
-        'bottom-left' |
-        'bottom-right' |
-        'left' |
-        'left-mid' |
-        'mid' |
-        'mid-mid' |
-        'right' |
-        'right-mid' |
-        'middle'
-    ), string>>;
-    truncate: string;
-    colors: boolean;
-    colWidths: number[];
-    colAligns: Array<"left" | "middle" | "right">;
-    style: Partial<{
-        'padding-left': number;
-        'padding-right': number;
-        head: string[];
-        border: string[];
-        compact: boolean;
-    }>;
-    head: string[];
+interface TableOptions<T> {
```

Comment:
Should this type parameter `T` have the same constraint as the `T` from `Table`? I think so, since something of type `TableOptions` will be used to construct a `Table`.

Reply:
fixed!